### PR TITLE
Docker updates

### PR DESCRIPTION
**Pull request by haitaohuang**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+linux/installer/docker
+linux/installer/docker/*
+docker
+docker/*
+docker/util/*
```
Comment (lzha101) [inline .dockerignore:5]: Is it docker/build/* ?

File: docker/build/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Files in this directory demonstrate how to build and install the SGX SDK and PSW
 
 ###  Prerequisites
 1. Install [Docker and Compose](https://docs.docker.com/) and configure them properly following their respective installation guide.
-2. Install [SGX out-of-tree driver](https://github.com/intel/linux-sgx-driver). **Note**: See below to run with the DCAP driver or an SGX capable kernel.
+2. Install [SGX driver](https://github.com/intel/SGXDataCenterAttestationPrimitives/driver/linux). **Note**: See below to run with the legacy non-FLC SGX driver.
```
Comment (jbdelcuv) [inline docker/build/README.md:9]: Please use consistent terminology. We use "SGX driver" here but "DCAP driver" below.
We may need a subsequent PR to have this terms aligned with the Installation Guide, where there's a proposal to use the terms Flexible Launch Control Driver and Legacy Launch Control Driver.

File: linux/installer/docker/README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Files in this directory demonstrate how to build and deploy SGX enclave applicat
 
 ###  Prerequisites
 1. Install [Docker and Compose](https://docs.docker.com/) and configure them properly following respective their installation guide.
-2. Install [SGX out-of-tree driver](https://github.com/intel/linux-sgx-driver). **Note**: See below to run with the DCAP driver or an SGX capable kernel.
+2. Install [SGX driver](https://github.com/intel/SGXDataCenterAttestationPrimitives/driver/linux). **Note**: See below to run with the legacy non-FLC SGX driver.
```
Comment (jbdelcuv) [inline linux/installer/docker/README.md:9]: Same comment on terminology as above.

File: .dockerignore (hunk 1 above)
Comment (haitaohuang) [reply .dockerignore:5]: yes, will fix

File: linux/installer/docker/README.md (hunk 3 above)
Comment (haitaohuang) [reply linux/installer/docker/README.md]: using those names for now

**Pull request merged**